projects/storefrontlib/src/lib/store-finder/store-finder.module.ts

```diff
@@ -32,7 +39,41 @@ import { StoreFinderCoreModule, UrlTranslationModule } from '@spartacus/core';
     BootstrapModule,
     SpinnerModule,
     UrlTranslationModule,
-    StoreFinderCoreModule
+    StoreFinderCoreModule,
+    ConfigModule.withConfig(<CmsConfig | LayoutConfig>{
+      cmsComponents: {
+        StoreFinderComponent: {
+          selector: 'cx-store-finder',
+          childRoutes: [
+            {
+              path: 'find',
+              component: StoreFinderSearchResultComponent
+            },
+            {
+              path: 'view-all',
+              component: StoreFinderStoresCountComponent
+            },
+            {
+              path: 'country/:country' /* 'country/:country/region/:region', */,
```

Comment:
Why there is the comment `/* 'country/:country/region/:region', */`?

Reply:
Thanks! Left comment to fix it later, but missed it somehow ;-)